packages/cli/src/commands/deploy.ts

```diff
@@ -26,8 +26,8 @@ type DeployCommandOptions = {
   verbose?: boolean;
 };
 
-type ManifestSequence = DeployManifest["sequences"][number];
-type ManifestStep = ManifestSequence["steps"][number];
+type ManifestJob = DeployManifest["jobs"][number];
```

Comment:
These types should come from the generated manifest types, no? Checkout how the test manifest uses reference typings in its json-schema.

Reply:
Hmm, the thing is that there is a special condition set for the `Workflow` types in the template scripts for those types (literally an `isWorkflow` condition). Even if we wanted to do that here as well (which I tried), we'd have to add another condition so that it only applies for version `0.2` of the DeployManifest, else it will error because version `0.1` did not use ref types. I considered adding that as well but it seems too hacky to me; maybe we could improve type exporting on the manifest packages in a separate PR

Reply:
okay no problem, sounds good!